README.md

```diff
@@ -6,7 +6,7 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 # Built With
 - React
-- Reduc
+- Redux
 
 # Getting Started
 
```

Comment:
- [ ] The screenshot in the README is broken. Please fix that to make the README professional.

Reply:
Will be fixed